sdks/java/extensions/sql/src/main/java/org/apache/beam/sdk/extensions/sql/meta/provider/bigquery/BigQueryTable.java

```diff
@@ -138,12 +138,7 @@ public BeamTableStatistics getTableStatistics(PipelineOptions options) {
       builder.withSelectedFields(fieldNames);
```

Comment:
Should be:
`builder = builder.withSelectedFields(fieldNames);`
Will fix in a different PR.

Reply:
I think this is fine as it is. Builder normally stores the changes and just returns itself as a convenience.

Reply:
Talked about this more, `builder` isn't actually a builder.